### DISCUSSION TITLE
Info about installed version in the WebUI GUI

### DISCUSSION THREAD

**Discussion by AleksanderKruecken** [monkeymademe/CamUI#74] [Ideas]
Hi, 

it seems I can't find info which version of Picamera2 WebUI I have installed on the GUI page (tabs Home, Camera info, About).
git git describe --tags command returns what I'm looking for. It would be nice If I would see this info on web GUI page.

**Comment by monkeymademe** [comment]
You are right I need to handle versions better to be honest. I'll put that on my list
